content/doc/book/using/using-jmeter-with-jekins.adoc

```diff
@@ -0,0 +1,158 @@
+It means you need to set all Test plans on JMeter as a first step 
+and then info to Jenkins which test it needs to call.
+
+Follow these steps to create a Test Plan:
+
+1. Run the file: `<YOUR-JMETER-PATH>>\bin\jmeter.bat` to open the JMeter GUI; (e.g.: C:\jmeter\bin\jmeter.bat in my case)
++
+[NOTE]
+====
+* To Linux user will be the jmeter.sh
+* In a definitive installation you can set these commands to your path system or System variables
+====
+2. From JMeter GUI go to `File menu` and then pick the `New option`;
+3. Fill a name to your test plan. (e.g.: Jenkins.io Test Plan )
+4. Now on the left side, click with the right button on your test plan item, go through this path `Add > Thread(users) > Thread Group`, and select it.
++
+image:jmeter/jmeter-04.png[plan test screen]
+5. Increase the `Number of threads (users)` to 5 at Thread Group and to 2 on the `Loop Count`;
+6. Now on the left side, click with the right button on your Thread Group item, go though: `Add > Sampler > Http Requester` and select Http Requester option;
++
+image:jmeter/jmeter-05.png[Thread Group]
+7. At Http Request, Fill: The `Name` of test, the `Server Name`, and the `Path Context`; (e.g: Installing; www.jenkins.io; and /doc/book/installing/ )
++
+image:jmeter/jmeter-06.png[Http Resquest sample]
+8. Repeat steps 6 and 7 more two times to different contexts/pages. (e.g.: www.jenkins.io/node)
+9. Now our plan has 3 things to test. 
+10. To add a visual report click with the right button on your Thread Group item, go though: `Add > Listener > View results in table` and select View results in table option;
++
+image:jmeter/jmeter-07.png[Save and run pan test]
+11. Now let's save our plan test. Click on "disk icon", give a name a save with a .jmx extension (e.g.: jenkins.io.jml)
+12. Now run the test and see the table results.
++
+image:jmeter/jmeter-08.png[test results]
+
+=== First JMeter Test Plan using terminal commands
+
+Our Test seems workly well on GUI, 
+but in order to integrate it on Jenkins, it needs to be able to run it by commands as well.
+
+Follow these steps to test:
+
+1. From the terminal run the command:
++
+[bash]
+----
+c:\jmeter\bin\jmeter -j jmeter.save.saveservice.output_format=xml -n -t c:\jmeter\bin\jenkins.io.jmx -l c:\jmeter\reports\jenkins.io.report.jtl
+----
+2. If everything works well the report file will be create at indicated location by -l paramter:
++
+image:jmeter/jmeter-09.png[manual test results]
+
+Now we have everything we need to create the Jenkins pipeline using Jmeter
+
+== Jekins and JMeter running together
+
+Now we have everything we need to execute JMeter from Jekins. Follow these steps:
+
+1. From Jenkins dashboard go to `New Item`;
+2. Fill the item name (e.g.: JmeterTest), select freestyle project, then press OK;
+3. Go to Build `Environment tab`, click on `Add build step`, and select the option: `Execute Windows batch command`
+4. Now fill the field with same code we use to run Jmeter at previous section:
```

Comment:
With this definition of the JMeter job, it will only run on the Jenkins controller (formerly known as the Jenkins master).  I think we should describe how to run on agents rather than running on the controller.  Did I misunderstand something in the instructions?
```suggestion
4. Now fill the field with same code we use to run Jmeter at previous section:
```